elfloader-tool/CMakeLists.txt

```diff
@@ -344,6 +344,8 @@ if(DEFINED KernelDTBPath)
     set_property(SOURCE src/drivers/driver.c PROPERTY OBJECT_DEPENDS ${DEVICES_GEN_H})
 endif()
 
+set_property(SOURCE src/common.c PROPERTY OBJECT_DEPENDS ${IMAGE_START_ADDR_H})
```

Comment:
Is this really necessary? With this PR `#include <image_start_addr.h>` is added to `common.c`, so it should be part of the normal dependency generation.